web/src/routes/(user)/sharing/+page.svelte

```diff
@@ -38,23 +38,23 @@
 		<LinkButton on:click={createSharedAlbum}>
 			<div class="flex place-items-center gap-1 text-sm">
 				<PlusBoxOutline size="18" />
-				Create shared album
+				<span class="max-xs:hidden">Create shared album</span>
```

Comment:
This removes an accessible name for screens smaller than breakpoint `xs`

Reply:
Fixed in 6baf73e0468a4ab5dffb89b639223c788e6f9d58